commit 0ad7a64d69cdbe520d0df80d100cd6b891cc12fb
Author: John Clements <john.clements@amd.com>
Date:   Mon Aug 3 14:24:50 2020 +0800

    drm/amdgpu: enable RAS support for sienna cichlid
    
    enabled GECC error injection and query support
    
    Reviewed-by: Guchun Chen <guchun.chen@amd.com>
    Signed-off-by: John Clements <john.clements@amd.com>
    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>

diff --git a/drivers/gpu/drm/amd/amdgpu/gmc_v10_0.c b/drivers/gpu/drm/amd/amdgpu/gmc_v10_0.c
index 35d21f330b0a..dd91a93de62a 100644
--- a/drivers/gpu/drm/amd/amdgpu/gmc_v10_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/gmc_v10_0.c
@@ -633,6 +633,10 @@ static int gmc_v10_0_late_init(void *handle)
 	if (r)
 		return r;
 
+	r = amdgpu_gmc_ras_late_init(adev);
+	if (r)
+		return r;
+
 	return amdgpu_irq_get(adev, &adev->gmc.vm_fault, 0);
 }